SafeguardSessions/modules/error/ErrorBase.pqm

```diff
@@ -1,7 +1,16 @@
 let
     ErrorBase.Type = Extension.ImportFunction("ErrorBaseType", "Types.pqm"),
-    ErrorBase.Implementation = (reason as text, message as text, detail as record) =>
-        error [Reason = reason, Message = message, Detail = detail],
+    ErrorBase.Implementation = (
+        reason as text, message as text, detail as record, optional manual as nullable logical

```

Comment:
Ack, but personally, I would prefer manuallyHandled as parameter name because manual alone does not tell what is the purpose of the variable. You would need to read through the code find out that reason,

Reply:
Done. ;)

Reply:
Thx. :)